Build.fsx

```diff
@@ -183,9 +184,7 @@ Target "CleanNuGetPackages" (fun _ ->
 )
 
 Target "NuGetPack" (fun _ ->
-    let version = "Src/AutoFixture/bin/Release/Ploeh.AutoFixture.dll"
-                  |> GetAssemblyVersion
-                  |> (fun v -> sprintf "%i.%i.%i" v.Major v.Minor v.Build)
+    let version = FileVersionInfo.GetVersionInfo("Src/AutoFixture/bin/Release/Ploeh.AutoFixture.dll").ProductVersion
```

Comment:
@ecampidoglio Thank you for this suggestion. I've improved code a bit, so now:
1. Use calculated NuGet version directly from the variable, rather than from assembly attribute.
2. AssemblyInformationalVersion attribute was changed to have the following format: `NuGetVersion-CommitSHA`. That would be very useful for the "nightly" builds as you will be able to quickly find the corresponding base commit. If we are on the tag now, SHA code isn't appended (because you can find commit by tag).

Let me know whether you agree with this change.